admin/data-collection.rst

```diff
@@ -152,6 +152,78 @@ example, you cannot use operations like ``less than`` or ``greater than`` on
 strings. If you select metric from the list using the :guilabel:`Select` button,
 correct data type will be set automatically.
 
+
+Units
+~~~~~
+
+For user convenience collected DCI values can have the following units assigned:

```

Comment:
Most of the units are just displayed after the value, but some of them are special and affect how collected data is displayed: 
- Epoch time - converts collected int to human readable timestamp
- Uptime - converts number of seconds since uptime to human readable format
- "%", "°C", "°F", "dbm" will not use multipliers 
- Evrything with (IEC) will use binary multipliers both for calculation and to display value, but  (IEC) and (Metric) will be removed when value is displayed.

All other values are just default values that are used as a postfix for value while displaying. This setting is taken into consideration only to display value, it is not converting value in the database. Selection here will be taken to format value when macro %<{format-specifier}name> with formatting is used.